courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2565 @@
+{
```

Comment:
<p>Do you need this block of code? The notebook ran fine for me without. I'd opt to leaving it out to avoid confusions with students.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>